proposals/lip-0057.md

```diff
@@ -870,7 +869,7 @@ unlockTransactionParams = {
 ```python
 def verify(trs: Transaction) -> None:
     trsParams = decode(unlockTransactionParams, trs.params)
```

Comment:
I think we can remove this line as well

Reply:
So this will be an empty function with just a `return` statement? Shouldn't we, then, just remove the whole function?

Reply:
Done